sky/skylet/attempt_skylet.py

```diff
@@ -2,22 +2,45 @@
 
 import os
 import subprocess
-import sys
 
 from sky.skylet import constants
 
+
+def restart_skylet(kill_old=False):
+    with open(constants.SKYLET_VERSION_FILE, 'w+') as v_f:
+        v_f.write(constants.SKYLET_VERSION)
+
+    if kill_old:
+        #pylint: disable=subprocess-run-check
+        subprocess.run(
+            'ps aux | grep "sky.skylet.skylet" | grep "python3 -m"'
+            '| awk \'{print $2}\' | xargs kill',
+            shell=True,
+        )
```

Comment:
```suggestion
        subprocess.run(
            'ps aux | grep "sky.skylet.skylet" | grep "python3 -m"'
            '| awk \'{print $2}\' | xargs kill',
            shell=True,
            check=False,
        )
```

Reply:
don't think we should check this because it fails if the skylet is not running

Reply:
Oh, this is just setting the `check` explicitly to `False` instead of having the `pylint: disable`

Reply:
ah i see you're right